features/katas/basic_gates/phase_change.feature.md

```
@@ -1,5 +1,5 @@
-# Feature: Quantum Katas BasicGates Task 1.6 PhaseChange
-  Task 1.6 PhaseChange: |1⟩ 成分に一般角の位相を掛ける
+# Feature: Quantum Katas BasicGates タスク 1.6 PhaseChange
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)